src/components/molecules/UserProfilePicture/UserProfilePicture.tsx

```diff
@@ -31,6 +33,7 @@ const UserProfilePicture: React.FC<UserProfilePictureProp> = ({
   setSelectedUserProfile,
   reactionPosition,
   user,
+  showNametags,
```

Comment:
Since this is an optional parameter, you can set a default value for it at this point, which can simplify later code.

```suggestion
  showNametags = true,
```

Reply:
done